lms/static/scripts/frontend_apps/components/test/LaunchErrorDialog-test.js

```diff
@@ -83,7 +83,7 @@ describe('LaunchErrorDialog', () => {
       retryAction = 'Try again',
     }) => {
       it(`displays expected error for "${errorState}" error state`, () => {
-        const error = new Error('Detailed error info');
```

Comment:
This now correctly gets printed to the DOM and affected this test. Simple fix is to just remove the error's message